src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
```

Comment:
It is better to set box-sizing for all elements(*) so it's the same for whole page